package.json

```diff
@@ -138,35 +138,23 @@
                 },
```

Comment:
The option to delete a (non-active) branch that has an upstream should still be available.

If you want, you could modify the name to be "delete local branch" or add a confirmation if the branch has an upstream.
I don't consider either of those extras necessary, but I do think delete should still be available.

Reply:
I just missed it, thanks for pointing out